packages/create-expo-app/src/analytics.ts

```diff
@@ -0,0 +1,189 @@
+import JsonFile from '@expo/json-file';
+import crypto from 'crypto';
+import fs from 'fs';
+import fetch from 'node-fetch';
+import os from 'os';
+
+import { getStateJsonPath } from './paths';
+import { getSession } from './sessionStorage';
+
+const packageJSON = require('../package.json');
+
+const xdlWriteKey = '1wabJGd5IiuF9Q8SGlcI90v8WTs';
+const analyticsEndpoint = 'https://cdp.expo.dev/v1/batch';
+const version = '1.0.0';
+const library = 'create-expo-app';
+
+//#region mostly copied from @expo/rudder-sdk-node https://github.com/expo/rudder-sdk-node/blob/main/index.ts
```

Comment:
If there were tests attached please copy them over too

Reply:
the code here is sufficiently different from what's in the library that we cannot just port the tests over. I'll add in a new test though.